backend/main.py

```
@@ -25,19 +27,29 @@
+            discovery_message.encode("utf-8"), broadcast_address, UDP_PORT
+        )
         await asyncio.sleep(5)
 
+
 @app.on_event("startup")
 async def startup_event():
     print("Starting WhisperNet core...")
     api.loop = asyncio.get_running_loop()
     core_lib.start_udp_listener(UDP_PORT, c_callback_handler)
-    asyncio.create_task(discover_peers_task())
+    discovery_task = asyncio.create_task(discover_peers_task())
```

Ruff (0.14.0)

Local variable `discovery_task` is assigned to but never used

Remove assignment to unused variable `discovery_task`

(F841)